content/blog/infrastructure-as-code-with-aws-cdk.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Infrastructure as Code with AWS CDK: A Success Story"
```

Comment:
This blog seems to be doing two things: introducing AWS CDK and discussing a real problem that was solved using CDK.

This makes it a little confusing because, at the start, we introduce the CDK concepts and then, in later stages, justify the choice to use AWS CDK based on requirements and what it provides.